cypress/utils/utils.ts

```diff
@@ -220,45 +223,45 @@ export function login(username?: string, password?: string, firstLogin = false):
     const sessionId = (username ?? "login") + (firstLogin ? "FirstLogin" : "");
 
     return cy.session(sessionId, () => {
-        cy.log("Login in");
-        cy.visit(Cypress.env("tackleUrl"), { timeout: 120 * SEC });
-        cy.wait(5 * SEC);
-        cy.get("h1", { timeout: 120 * SEC, log: false }).then(($title) => {
-            // With auth disabled, login page is not displayed and users are taken straight
-            // to the Application Inventory page.
-            if (
-                $title.text().toString().trim() !== "Sign in to your account" &&
-                $title.text().includes("Application inventory")
-            ) {
-                return;
-            }
+        cy.log("Log in");
+        cy.visit("/");
```

Comment:
```suggestion
        cy.visit("/", {timeout: 120 * SEC});
```

The time Cypress will wait for this `cy.visit` to complete is limited by the `pageLoadTimeout` default value which is 6 secs, There are many cases where the login page can take more than that to load due to different causes, i.e on a fresh installation of konveyor or in tests that restart pods.

Reply:
Let me know what y'all think of the latest changes

Reply:
Looks perfect now, thanks!